tests/extras/fastapi/test_fastapi.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Tests that a pipeline with the FastAPI app runs correctly,
+and is able to consume and produce messages.
```

Comment:
I  realise that it's only a test file, but module docstrings are supposed to start with a 1 line description.